tests/integration/seed/seedsVerification.test.js

```diff
@@ -1994,6 +1994,13 @@ describe('SEEDS VERIFICATION', () => {
           );
           expect(doUsersHaveRoleInWrongInterface).toBeFalsy();
         });
+
+        it('should pass if every user has only one of the two fields firstMobileConnection or loginCode', () => {
```

Comment:
ce nom porte un peu a confusion : on dirait qu'il faut au moins l'un des deux...
que penses tu de mettre "should pass if every user has either firstMobileConnection or loginCOde or none of these 2 fields"

Reply:
ok